skycoin/skywire
# Skip auto-transport when transport available

Did you run `make format && make check`? Yes

Fixes #1297 

 Changes:	
- add a condition at the beginning of `getRouteSetupHooks` method (a.k.a. autotransport) to skip whole of process if a transport from visor to vpn-server be available. 

How to test this PR:
- `make build`
- run a visor (in debug log-level), and run a vpn-server (you can use `036c0e89735c675abce702a3d46ad1bf736bd8236dbd734bfaf368959de2c2b967`)
- create manual transport in `dmsg` or `sudph` to vpn-server
- trying to run vpn-client
- should see this log on visor logs `Debug [router]: Established transport exist. Type: dmsg`, and no trying for new transport in order `stcpr`>`sudph`>`dmsg`
- also vpn-client should running on exist transport, so check Routing tab of visor details
